src/panels/lovelace/editor/card-editor/hui-badge-preview.ts

```diff
@@ -0,0 +1,99 @@
+import "@polymer/paper-input/paper-textarea";
+
+import { HomeAssistant } from "../../../../types";
+import { LovelaceBadgeConfig } from "../../../../data/lovelace";
+import { ConfigError } from "../types";
+import { computeRTL } from "../../../../common/util/compute_rtl";
+import { LovelaceBadge } from "../../types";
+import { createBadgeElement } from "../../create-element/create-badge-element";
+import { createErrorBadgeConfig } from "../../badges/hui-error-badge";
+
+export class HuiBadgePreview extends HTMLElement {
+  private _hass?: HomeAssistant;
+  private _element?: LovelaceBadge;
+  private _config?: LovelaceBadgeConfig;
+
+  private get _error() {
+    return this._element?.tagName === "HUI-ERROR-BADGE";
```

Comment:
I'm not sure how to determine if its an error. May need to create a public error on the state label badge.

OR 

do we need to do anything special for errors on badges? Maybe just checking if the entity exists?

Reply:
It would be a `hui-error-card` as that is handled in `create-element-base`

Reply:
![image](https://user-images.githubusercontent.com/5662298/77531380-1af91200-6e93-11ea-9d52-f640a37b8522.png)

Reply:
It looks like we do have an `error-badge` but it is currently unused.